recipes/diffpy.srfit/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "diffpy.srfit" %}
+{% set version = "3.1.0" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/d/diffpy_srfit/diffpy_srfit-{{ version }}.tar.gz
+  sha256: cd1bae4e8dddc36ac5a69334678de3d25bcbd97b6cde76006a01067c27166a13
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - setuptools-git-versioning >=2.0
+    - pip
+
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - matplotlib-base
+    - scipy
+
+test: 
+  requires:
+    - python {{ python_min }} 
+    - pip
+    - pytest
+    - sympy
+  source_files:
+    - tests
+  commands:
+    - pip check
+    - pytest
+
+about:
+  home: https://github.com/diffpy/diffpy.srfit/
+  license: Free To Use But Restricted
```

Comment:
Can you use `LicenseRef-diffpy` instead? It would be nice to standardized the others onez that are already in conda-forge as well to make then easily discoverable.

Reply:
I have changed this to `LicenseRef-diffpy` as requested and added a statement that the license file is 100% compatible with BSD-3 as you requested.